src/docs/index.md

```diff
@@ -0,0 +1,151 @@
+# Simple Standard for Sharing Ontological Mappings (SSSOM)
+
+![SSSOM banner](images/sssom-banner.png)
+
+**SSSOM** is the Simple Standard for Sharing Ontological Mappings. It comprises three distinct components that are intended to be used together to facilitate the exchange of semantic mappings:
+
+1. a machine-readable and extensible vocabulary to describe metadata of mappings;
+2. a data model to represent mappings and their associated metadata;
+3. several file formats to represent sets of mappings on disk and on the network.
+
+Beyond defining the standard itself, the **SSSOM Core Team** also aims to implement open and community-driven collaborative workflows designed to evolve the standard continuously to address changing requirements and mapping practices, and to provide reference tools and software libraries for working with the standard.
+
+## SSSOM at a glance
+
+### Basic concepts
+
+The [data model](spec-model.md) of SSSOM is centered around two fundamental concepts: mappings and mapping sets.
+
+A **SSSOM mapping** is a statement that there is a correspondence of some sort between two semantic entities. It comprises two components:
+
+1. The **core mapping** (or **raw mapping**), which is a triple `<subject, predicate, object>` that represents the correspondence itself between a subject entity, for example a class in an ontology, and an object entity, for example an identifier in some database, via a semantic mapping predicate, for example `skos:exactMatch`.
+2. **Metadata** that provide supplementary pieces of information about the core mapping. This notably includes information pertaining to the *provenance* of the statement (for example, who emitted the statement, in other words who decided that the subject and the object should be mapped) and its *justification* (why should the subject and the object be mapped).
+
+A **SSSOM mapping set** is a collection of SSSOM mappings, with its own metadata.
+
+### The SSSOM/TSV format
+
+The main format proposed by the SSSOM standard to exchange mapping sets is the [SSSOM/TSV format](spec-formats-tsv.md). Here is a basic example of a file in that format:
+
+```
+#curie_map:
+#  HP: "http://purl.obolibrary.org/obo/HP_"
+#  MP: "http://purl.obolibrary.org/obo/MP_"
+#license: "https://creativecommons.org/publicdomain/zero/1.0/"
+#title: "Sample mapping set"
+subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification
+HP:0008551	Microtia	skos:exactMatch	MP:0000018	small ears	semapv:LexicalMatching
+HP:0000411	Protruding ear	skos:exactMatch	MP:0000021	prominent ears	semapv:LexicalMatching
```

Comment:
No objection, but:

1) I will use a simplified version of that example (with less metadata). The point in the “at a glance” section is to have a _basic_ example to give the reader an idea of what the format looks like – it does not have to showcase the various metadata slots.

2) I will leave aside the mapping that uses the `sssom:NoMapping` special identifier, which, unless I have missed something, is not specified anywhere (and the IRI does not resolve to anything). I know that special identifier has been discussed in #28, but (a) in the end the agreement was to use `sssom:NoTermFound` rather than `sssom:NoMapping`, and (b) even after the agreement was reached at the end of the discussion, the special identifier did _not_ find its way into the spec.

If that special identifier is still a desirable feature (a question that is worth asking in my opinion, since the aforementioned discussion is almost three years old and seemingly nobody has complained that the special identifier was still missing), then I’d suggest that it be added to the `spec-model.md` document after this PR has been merged.

3) Less important, but that example is invalid as it is using `orcid:…` and `wikidata:…` CURIEs without declaring the `orcid` and `wikidata` prefix names. We should _really_ pay attention and make sure that we are giving examples that would _not_ be rejected by our own tools!

```
Invalid SSSOM data in file foodieinc-food.sssom.tsv: Some prefixes are undeclared: orcid, wikidata
```

Reply:
Also, that comment at the mapping set level is weird:

```
#comment: We could map to FOODON:00004187 instead which more specifically refers to
#  "raw" Pink apples. Decided against to be consistent with other mapping choices.
```

Looks like a duplicate of the mapping-level comment in the `KF_FOOD:0003 -- skos:exactMatch -- FOOD:00004186` mapping, and almost certainly the result of some kind of error.

Reply:
Looks great! Thank you! 

> Also, that comment at the mapping set level is weird:

Looks like an oversight, agree.